src/main/java/seedu/kitchenhelper/command/CookRecipeCommand.java

```diff
@@ -110,6 +111,9 @@ public int checkIfIngredientExpired(Ingredient ingredientToDeduct, int totalCook
                     totalCookedQty = 0;
                 }
             }
+            if (ingredientToDeduct.getQuantity() == 0) {
+                ingredientsList.remove(ingredientToDeduct);
+            }
```

Comment:
Great implementation to help clear the ingredient list from repeated ingredients, which makes it neater:)